### ISSUE TITLE
PEN as default TOOL don't work

### ISSUE THREAD

**Issue by HimonRana** [scaleflex/filerobot-image-editor#288]
You cannot use/draw **pen** when you have ANNOTATE as default TAB and PEN as default TOOL:

```
tabsIds={[TABS.ANNOTATE, TABS.ADJUST, TABS.RESIZE]}
defaultTabId={TABS.ANNOTATE}
defaultToolId={TOOLS.PEN}
```
Haven't tried other default TAB/TOOL but pen does not work as default. 

**_Workaround:_** You can click on any other tool and then back to pen to make it work.

Any fix for this soon?

**Comment by HimonRana** [comment]
Also PEN do not always get a selection after using it. Seems like if you draw fast with the pen it doesn't get a selection. We need and improvment on this.

**Comment by AhmeeedMostafa** [comment]
@HimonRana we are currently a bit busy to check, so we will consider checking and fixing it if found in the future once we have time.

**Comment by AhmeeedMostafa** [comment]
The issue is fixed in version 4.3.8, please re-check it and let us know if all is good.
